dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -264,7 +264,7 @@ def test_profiled_mode(self):
         df = pd.Series([1, 1, 1, 1, 1, 1, 1]).apply(str)
         profiler = FloatColumn(df.name)
         profiler.update(df)
-        self.assertAlmostEqual(1, profiler.mode, places=2)
+        np.testing.assert_array_almost_equal([1], profiler.mode, decimal=2)
```

Comment:
what is the value such that this is not just `1`

Reply:
Could you clarify your question?

Reply:
why are we having to assert almost equal when the mode should be `[1]`

`self.assertListEqual([1], profiler.mode)`

Reply:
The mode is estimating it off the histogram, so I believe it's not exactly equal to one as the bin it falls in is 1.00, 1.002 or something similar.

Reply:
in this case, i believe it should be exact since just 1 value in the column. Otherwise might be different.

I just find the `assert_array_almost_equal` strange as it is setting a tolerance around it which is variable to the size of the input values and not normalized.

Reply:
thoughts?

Reply:
for this case specifically with unique values, it is exactly 1, but for other cases, using almost equal is fine to me.